### PR TITLE
Add a new blog

### PR DESCRIPTION
**Pull request by Ploypiti**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Add a new blog & image

### CODE REVIEW

**Review by ParthS007 — COMMENTED** [review]
@Ploypiti Please fix #243 also with this and provide a preview link.

**Comment by Ploypiti** [comment]
@ParthS007 Strange, It worked just well on my local editor but it didn't work with the preview.

File: index.html [hunk 1]
```diff
@@ -407,7 +407,7 @@ <h4 class="uppercase mb16">Blogs</h4>
 		{% for blog in site.data.blogs %}
 			<div class="col-md-3 col-sm-6">
 				<a href="{{blog.url}}" target="_blank">
-					<div class="image-tile outer-title text-center">
+					<div class="image-tile outer-title text-center" style="margin-bottom: 0px;">
```
Comment (ParthS007) [inline index.html]: @Ploypiti Please don't use inline styling, it is not considered as a good practice. Please read more about it [here](https://www.thoughtco.com/avoid-inline-styles-for-css-3466846)

**Comment by ParthS007** [comment]
@Ploypiti Have you pushed to gh-pages properly?

**Comment by Ploypiti** [comment]
Yes, I think.

**Comment by Ploypiti** [comment]
@ParthS007 Can you please merge this first as I will solve the blog in another PR, I need to install msys2 first and it takes 3-6 hours (according to the downloader, I'm not sure).

**Review by anshumanv — APPROVED** [review]

**Review by the-codingninja — APPROVED** [review]

**Pull request merged**